source/Mission.h

```diff
@@ -212,6 +217,8 @@ class Mission {
 	// Parameters for generating random passenger amounts:
 	int passengerLimit = 0;
 	double passengerProb = 0.;
+	// Escort payment
+	int cost = 0;
```

Comment:
The name of the variable could be clear enough not to require a comment; for instance `escortSalary` instead of cost

Reply:
Will do this

Reply:
Is done